packages/api/src/beacon/routes/debug.ts

```diff
@@ -17,14 +17,12 @@ import {
   ContainerData,
 } from "../../utils/index.js";
 import {HttpStatusCode} from "../../utils/client/httpStatusCode.js";
-import {ApiClientResponse} from "../../interfaces.js";
+import {MEDIA_TYPES, parseAcceptHeader} from "../../utils/server/parseAcceptHeader.js";
+import {ApiClientResponse, ResponseFormat} from "../../interfaces.js";
 import {ExecutionOptimistic, StateId} from "./beacon/state.js";
 
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
-export type StateFormat = "json" | "ssz";
-export const mimeTypeSSZ = "application/octet-stream";
```

Comment:
I like what you did here by moving this to `MEDIA_TYPES`.  There is a similar declaration in `packages/api/src/beacon/routes/debug.ts`. Would you be interested in swapping that as well to use `MEDIA_TYPES`?

https://github.com/ChainSafe/lodestar/blob/1d14dc1229f8462adfb3b947d37aa1c6bdc3c5a7/packages/api/src/beacon/routes/debug.ts#L26

https://github.com/ChainSafe/lodestar/blob/1d14dc1229f8462adfb3b947d37aa1c6bdc3c5a7/packages/api/src/beacon/routes/debug.ts#L152-L153

Reply:
Already removed as part of this PR:

```
$ git grep mimeTypeSSZ | wc -l
0
```

Reply:
yup! you did catch it in both locations.  great!